compute-budget-instruction/src/compute_budget_instruction_details.rs

```diff
@@ -18,8 +18,7 @@ use {
 };
 
 #[cfg_attr(test, derive(Eq, PartialEq))]
-#[cfg_attr(feature = "dev-context-only-utils", derive(Clone))]
-#[derive(Default, Debug)]
+#[derive(Clone, Default, Debug)]
```

Comment:
Why did this become `Clone` for non-test/bench?

Reply:
Keeping it under `dev-context-only-utils` leads to this error:
```
error[E0277]: the trait bound `solana_compute_budget_instruction::compute_budget_instruction_details::ComputeBudgetInstructionDetails: Clone` is not satisfied
  --> runtime-transaction/src/transaction_meta.rs:41:5
   |
35 | #[cfg_attr(feature = "dev-context-only-utils", derive(Clone))]
   |                                                       ----- in this derive macro expansion
...
41 |     pub(crate) compute_budget_instruction_details: ComputeBudgetInstructionDetails,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Clone` is not implemented for `solana_compute_budget_instruction::compute_budget_instruction_details::ComputeBudgetInstructionDetails`
```

Adding the following as a `dev-dependency` in `runtime-transaction` also doesn't help, due to the use in `transaction_meta`.
```
solana-compute-budget-instruction = { workspace = true, features = ["dev-context-only-utils"] }
```

I also tried updating the `dependency` to the above statement, but that fails because it's not an `optional` dep.

I can try other suggestion though, as I understand the current change is not ideal.

Reply:
What check command were you running? I usually run `./cargo nightly clippy --all-targets`.
Your branch compiles fine if I just move it back to the dev-context-only-utils flag.

Reply:
This seems to work:

under `dev-dependencies`:
```
solana-compute-budget-instruction = { workspace = true, features = [
    "dev-context-only-utils",
] }
```

AND change `features` section:
```
[features]
dev-context-only-utils = [
    "solana-compute-budget-instruction/dev-context-only-utils",
]
```

Reply:
Thanks! I'll try it and update the PR.

Reply:
Thanks @apfitzge, that worked. Updated the PR.